src/test/java/uk/gov/hmcts/reform/juddata/camel/processor/ArchiveAzureFileProcessorTest.java

```diff
@@ -38,8 +64,13 @@ public void test_process() throws Exception {
                 .getClass().getDeclaredField("archivalDateFormat"), "dd-MM-yyyy--HH-mm");
         FieldSetter.setField(azureFileProcessor,azureFileProcessor
                 .getClass().getDeclaredField("fileReadTimeOut"), 1000);
-        exchange.setProperty("CamelLoopIndex", 0);
-        azureFileProcessor.process(exchange);
-        Assert.assertNotNull(exchange.getIn().getHeader("fileName"));
+        FieldSetter.setField(azureFileProcessor, azureFileProcessor
+                .getClass().getDeclaredField("leafRouteName"), "direct:leaf");
+        FieldSetter.setField(azureFileProcessor, azureFileProcessor
+                .getClass().getDeclaredField("leafArchivalFileNames"), archivalFileNames);
+
+        //exchange.setProperty("CamelLoopIndex", 0);
```

Comment:
remove the commented code

Reply:
we are not using verify() method  in the test even though mock used .This will raise again as part of QA concern

Reply:
assertNot null is there. I think it won't be issue. Any way we are just mocking this for test coverage

Reply:
Guys for the method where we are checking the behavior or  the logic within the method then  we should use verify... doing just an assert to check null/not null is not sufficient